src/cedar.js

```diff
@@ -978,20 +985,28 @@ Cedar._mixin = function(source) {
  * @param  {String}   url      URL to json file
  * @param  {Function} callback node-style callback function (error, data)
  */
-Cedar.getJson = function( url, callback ){
+Cedar.getJson = function( url, callback, timeout ){
```

Comment:
why not use `self._timeout` here instead of taking `timeout` as an argument? The way you have it, the timeout will only be aplied for calls to `update()` not calls to fetch sepcification, etc..